rotkehlchen/data/importer.py

```diff
@@ -1435,3 +1459,122 @@ def import_bisq_trades_csv(self, filepath: Path, **kwargs: Any) -> Tuple[bool, s
                 except KeyError as e:
                     return False, str(e)
         return True, ''
+
+    @staticmethod
+    def _group_binance_rows(
+        rows: List[BinanceCsvRow],
+        **kwargs: Dict[str, Any],
+    ) -> Tuple[int, Dict[Timestamp, List[BinanceCsvRow]]]:
+        multirows: Dict[Timestamp, List[BinanceCsvRow]] = defaultdict(lambda: [])
+        skipped_count = 0
+        formatstr = kwargs.get('timestamp_format')
+        for csv_row in rows:
+            try:
+                timestamp = Timestamp(deserialize_timestamp_from_date(
+                    date=csv_row['UTC_Time'],
+                    formatstr=formatstr if formatstr is not None else '%Y-%m-%d %H:%M:%S',  # type: ignore  # noqa: E501
```

Comment:
For every importer we inject user configurations. At the moment the only one we provide the timestamp because some locations change them depending on location